components/inputs/docs/input-date-time.md

```diff
@@ -69,7 +108,7 @@ To make your usage of `d2l-input-date` accessible, use the following properties
 
 ## Date Range Input [d2l-input-date-range]
 
-The `<d2l-input-date-range>` component consists of two input-date components - one for the start of a range and one for the end of a range. Values specified for these components (through the `start-value` and/or `end-value` attributes) are displayed if specified, and selected values are reflected.
+Use the `<d2l-input-date-range>` component when users need to enter two dates in a range, like a course start and end date. The component consists of two input-date components - one for the start of a range and one for the end of a range. Values specified for these components (through the `start-value` and/or `end-value` attributes) are displayed if specified, and selected values are reflected.
```

Comment:
I think the `Use <component> when ...` statements will be very useful for other devs!